tuna/miopen/yaml_files/sample1.yaml

```diff
@@ -26,6 +26,9 @@ import_configs:
   framework: 'Pytorch'
   fw_version: 1
 
+load_job:
+  enabled: true
```

Comment:
have you tested this? i think there are some other arguments required in load_job like...reason and such, maybe some fin steps

Reply:
tested this file and its working so far

Reply:
label is a required field for load_jobs, but up in the arg parsing it has a default value of 'new', perhaps that should be removed and a label added here in the sample.yaml